### PR TITLE
add address validation

### PR DESCRIPTION
**Pull request by 0xys**

### CODE REVIEW

**Review by avcdsld — CHANGES_REQUESTED** [review]
Let's do the arraying of error messages at this time.

File: frontend/src/components/BatchTransfer.tsx [hunk 1]
```diff
@@ -20,108 +20,110 @@ import { userAccountState } from '../store';
...
     );
     setTotalAmount(totalAmount);
+
     const remaining = new BigNumber(
       userAccount?.balance[currency.symbol] || 0
     ).minus(totalAmount);
     setRemaining(remaining);
 
+    const addressErrors: ValidationError[] = []
+    for(let i=0; i<outputs.length;i++) {
```
Comment (avcdsld) [inline frontend/src/components/BatchTransfer.tsx]: Just a comment.
Perhaps we should introduce a formatter with `.prettier` file.

File: frontend/src/components/BatchTransfer.tsx [hunk 2]
```diff
@@ -20,108 +20,110 @@ import { userAccountState } from '../store';
...
+
 const BatchTransfer = () => {
   const {
     handleSubmit,
     formState: { errors, isSubmitting },
   } = useForm();
   const [userAccount, setUserAccount] = useRecoilState(userAccountState);
   const [currency, setCurrency] = useState(FLOWCurrency);
-  const [recipientTemplate, setRecipientTemplate] = useState(''); // text area
+  const [txTemplate, setTxTemplate] = useState(''); // text area
```
Comment (avcdsld) [inline frontend/src/components/BatchTransfer.tsx]: This variable means information about the recipients and the amounts. The original name is not good, but `txTemplate` is also a bit strange.

**Review by avcdsld — APPROVED** [review]

File: frontend/src/components/BatchTransfer.tsx (hunk 2 above)
Comment (0xys) [reply frontend/src/components/BatchTransfer.tsx]: txTemplate -> outputsTemplate
https://github.com/flow-japan/batch-token-transfer-tool/pull/3/commits/5681064770206e23da7c75ac8a79f773f583b019

File: frontend/src/components/BatchTransfer.tsx (hunk 1 above)
Comment (0xys) [reply frontend/src/components/BatchTransfer.tsx]: did `prettier . --write`
https://github.com/flow-japan/batch-token-transfer-tool/pull/3/commits/846e070782fe0cbc0864b241d7d2f5fc118498c1

**Comment by avcdsld** [comment]
LTGM. Thanks!

**Pull request merged**